doc/rippled-example.cfg

```diff
@@ -972,6 +974,11 @@ pool.ntp.org
 [ips]
 r.ripple.com 51235
 
+# Absolute path or name of file containing validation quorum and
```

Comment:
How about

    # File containing validation quorum and trusted validator keys. Unless the
    # name specifies an absolute path, it will be considered relative to the
    # folder in which the configuration file is located.